README.md

```diff
@@ -29,3 +29,126 @@ OpenEBS/project
 ```
 
 > Note: Developer should keep `integration_test` completely decoupled from the rest of the project packages.
+
+## Instantiation
+
+Developer has to instantiate CITF using `NewCITF(configFilePath)` which will initialize the CITF instance with all the required configurations. 
+
+CITF has four components:- 
```

Comment:
CITF struct has four fields. I think if we talk about components CITF has only two (`environment` and `utils`)